services/core-web/src/tests/components/dashboard/varianceHomePage/__snapshots__/VarianceHomePage.spec.js.snap

```diff
@@ -69,28 +69,6 @@ exports[`VarianceHomePage renders properly 1`] = `
               ],
             }
           }
-          filterVarianceStatusOptions={
```

Comment:
should this have been removed?

Reply:
fixed. that constant doesn't exist anymore, used the new one.